src/mono/wasm/debugger/DebuggerTestSuite/EvaluateOnCallFrameTests.cs

```diff
@@ -1220,5 +1220,34 @@ public async Task EvaluateLocalObjectFromAssemblyNotRelatedButLoaded()
                 expected_props = new [] { TString("S"), TString("T"), TString("R") };
                 await CheckProps(props, expected_props, "props#2");
              });
+
+        [Fact]
+        public async Task EvaluateDefaultInterfaceMethod() =>  
+            await CheckInspectLocalsAtBreakpointSite(
+            "DefaultInterfaceMethod", "Evaluate", 7, "Evaluate",
+            "window.setTimeout(function() { invoke_static_method ('[debugger-test] DefaultInterfaceMethod:Evaluate'); })",
+            wait_for_event_fn: async (pause_location) =>
+            {
+                var id = pause_location["callFrames"][0]["callFrameId"].Value<string>();
+
+                await EvaluateOnCallFrameAndCheck(id,
+                    ("defaultFromIDefault", TString("DefaultMethod() from IDefaultInterface")),
+                    ("defaultFromIOverride", TString("DefaultMethod() from IDefaultInterface")),
+                    ("default2FromIOverride", TString("DefaultMethod2() from IOverrideDefaultInterface"))
```

Comment:
This is essentially checking whether local strings in a regular static method can be evaluated. And that those got populated by DIMs.
I would suggest trying to evaluate locals, or members when
- paused in a DIM. And maybe inspecting a previous frame.
- paused in a method called from a DIM, and inspecting the DIM frame

And async variations where the DIM is an async method.

Reply:
I think you mentioned somewhere to check that we can break, inspect, and resume from a DIM method, and that definitely is a good idea.